### ISSUE TITLE
False positive: `schema.core/def` is not linted correctly

### ISSUE THREAD

**Issue by sadikneipp** [borkdude/clj-kondo#605]
**version**
`clj-kondo v2019.11.07`

**platform**
OSX

**editor**
VSCode, Clojure Lint with `clj-kondo` installed through homebrew

**problem**
`schema.core/def` is not linted correctly

**repro**
```
(ns clojure-noob.core
  (:require [schema.core :as s]))

(s/def foo :- long "a long" 2)
```

**expected behavior**
No linter complaints.

**actual behaviour**
`src/clojure_noob/core.clj:4:8: error: unresolved symbol foo`

**Comment by borkdude** [comment]
Thank you!

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/44e36d102c0601cf1e21af2358c9ddddd2e1dca8
